kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Coordinator.java

```diff
@@ -64,6 +68,8 @@ public class Coordinator extends Channel implements AutoCloseable {
   private static final String COMMIT_ID_SNAPSHOT_PROP = "kafka.connect.commit-id";
   private static final String VTTS_SNAPSHOT_PROP = "kafka.connect.vtts";
   private static final Duration POLL_DURATION = Duration.ofMillis(1000);
+  private static final String TXID_VALID_UNTIL_PROP = "txid_valid_until";
```

Comment:
you two may be right...using 'until' seems weird. Should we change this to txid_valid_through? I can update the mirror code if we want to change this

Reply:
yeah, I like txid_valid_through better... I'll change it to use that in this PR.